docs-v2/guides/custom.mdx

```diff
@@ -269,18 +269,18 @@ To develop _actions_ locally and test them, run the following within `./nango-in
 ```bash
 nango dev # Continuously watches integration files for changes.
 ```
-Nango now watches your `nango-integrations` folder for changes and compiles the _action_ scripts & data models as needed. If there are any compilation errors (e.g. due to type issues), you can see them in the terminal where `nango dev` runs. 
+Nango now watches your `nango-integrations` folder for changes and compiles the _action_ scripts & data models as needed. If there are any compilation errors (e.g. due to type issues), you can see them in the terminal where `nango dev` runs.
 
 Fill in the `runAction` method with your integration code:
 
 ```ts slack-alert.ts
-import { NangoSync, SlackAlertResponse } from './models';
+import { NangoAction, SlackAlertResponse } from './models';
```

Comment:
Can the changes to the file only be isolated to the content changes? Looks like a lot of white space adjustments in the diff

Reply:
Sure Khaliq, updating my pull request.